### PR TITLE
Updated migration documentation

### PR DESCRIPTION
**Pull request by slewis74**
Depends on OctopusDeploy/Octopurls#109

### CODE REVIEW

File: migration-guide.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Migration Guide
+
+## v1 to v3
+
+The primary change between version 1 and 3 of this action is that v1 installs the C# CLI (`octo`) and v3 installs the [Go CLI](https://g.octopushq.com/ExternalToolOctopus) (`octopus`).
```
Comment (benPearce1) [inline migration-guide.md:5]: this link is not currently working, is this expected?

File: README.md [hunk 2]
```diff
@@ -53,9 +57,9 @@ Here's an example of invoking the `account list` command after installing the Oc
 
 The following input is optional:
 
-| Name      | Description                                                                      |   Default    |
-| :-------- | :------------------------------------------------------------------------------- | :----------: |
-| `version` | The version number of the Octopus CLI to download and install (i.e. `0.8.0`).    | `*` (latest) |
+| Name      | Description                                                                   |   Default    |
+| :-------- | :---------------------------------------------------------------------------- | :----------: |
+| `version` | The version number of the Octopus CLI to download and install (i.e. `0.8.0`). | `*` (latest) |
```
Comment (benPearce1) [inline README.md:62]: a link to the releases might be handy here, but not necessary

File: migration-guide.md (hunk 1 above)
Comment (slewis74) [reply migration-guide.md:5]: it won't until the dependent PR merges :)

**Review by benPearce1 — APPROVED** [review]
One

**Pull request merged**